docs/windows/README.md

```diff
@@ -0,0 +1,473 @@
+   - Double-click `PinShare-Setup.msi`
+   - Click "Next" through the installation wizard
+   - Accept the license agreement
+   - Choose installation directory (default: `C:\Program Files\PinShare`)
+   - Click "Install"
+
+3. **Complete installation**
+   - The installer will:
+     - Install all required binaries
+     - Create data directories
+     - Install and start the Windows service
+     - Add system tray application to startup
+
+4. **First launch**
+   - The PinShare service should start automatically
+   - Look for the PinShare icon in your system tray (bottom-right corner)
+   - Click "Open PinShare UI" to access the web interface
+
+## Getting Started
+
+### Accessing the UI
+
+After installation, access PinShare through:
+
+1. **System Tray**
+   - Right-click the PinShare icon
+   - Select "Open PinShare UI"
+
+2. **Browser**
+   - Navigate to: http://localhost:8888
+
+3. **Start Menu**
+   - Start Menu → PinShare → Open PinShare UI
+
+### First-Time Setup
+
+When you first open PinShare:
+
+1. The IPFS repository will be initialized automatically
+2. PinShare will generate a unique identity key
+3. You can start uploading files immediately
+
+### Uploading Files
+
+1. Click "Upload" or drag files to the upload area
+2. Files are automatically:
+   - Scanned for malware (if configured)
+   - Added to IPFS
+   - Shared with peers via libp2p
+
+## Configuration
+
+### Default Settings
+
+PinShare uses these default settings:
+
+| Setting | Default Value | Description |
+|---------|---------------|-------------|
+| UI Port | 8888 | Web interface port |
+| API Port | 9090 | Backend API port |
+| IPFS API | 5001 | IPFS daemon API port |
+| IPFS Gateway | 8080 | IPFS HTTP gateway |
+| IPFS Swarm | 4001 | IPFS P2P port |
+| libp2p Port | 50001 | PinShare P2P port |
+
+### Changing Configuration
+
+#### Method 1: Registry Editor (Advanced)
+
+1. Press `Win + R`, type `regedit`, press Enter
+2. Navigate to: `HKEY_LOCAL_MACHINE\SOFTWARE\PinShare`
+3. Modify values:
+   - `UIPort` - Change web UI port
+   - `OrgName` - Your organization name
+   - `GroupName` - Your group name
+   - `SkipVirusTotal` - 1 to skip virus scanning
+   - `EnableCache` - 1 to enable caching
+
+4. Restart the service:
+   ```cmd
+   net stop PinShareService
+   net start PinShareService
+   ```
+
+#### Method 2: Configuration File
+
+Edit: `C:\ProgramData\PinShare\config.json`
+
+```json
+{
+  "ui_port": 8888,
+  "pinshare_api_port": 9090,
+  "org_name": "MyOrganization",
+  "group_name": "MyGroup",
+  "skip_virus_total": true,
+  "enable_cache": true
+}
+```
+
+Then restart the service.
+
+### Data Directories
+
+PinShare stores data in:
+
+```
+C:\ProgramData\PinShare\
+├── config.json          # Configuration file
+├── logs\
+│   ├── service.log      # Service logs
+│   ├── ipfs.log         # IPFS logs
+│   └── pinshare.log     # Backend logs
+├── ipfs\                # IPFS repository
+├── pinshare\
+│   ├── pinshare.db      # SQLite database
```

Comment:
Let's check whether the sqlite integration is present in this branch - I suspect that it's likely not. If that's the case, let's remove this (and other references) to sqlite for now.